qubes-rpc/qvm-open-in-vm

```diff
@@ -64,6 +64,21 @@ if [ -z "$target" ] || [ -z "$filename" ]; then
     usage
 fi
 
+if [ -d "$filename" ]; then
+    printf 'Error! "%s" is a directory.\n' "$filename"
+    usage
+elif [ -L "$filename" ]; then
```

Comment:
Why reject symlinks? They open just fine (but when saving modified file, symlink is replaced with a copy of the file, and the actual target file remains unchanged).

Reply:
> Why reject symlinks? They open just fine (but when saving modified file, symlink is replaced with a copy of the file, and the actual target file remains unchanged).

I was not aware of that. Symlinks are now allowed in the amended PR.